src/app/mandate.model.ts

```diff
@@ -13,12 +16,8 @@ export interface Mandate {
   },
   numberOfMatchedDonations: number,
   giftAid: boolean,
-  donationAmount: {
-    "amountInPence": number,
-    "currency": "GBP"
-  },
-  matchedAmount: {
-    "amountInPence": number,
-    "currency": "GBP"
-  }
+  donationAmount: Money,
+  matchedAmount: Money,
+  totalCharityReceivesPerInitial: Money,
+  giftAidAmount: Money,
```

Comment:
wondering if giftAidAmount (and I guess matchedAmount) should be optional / nullable. Maybe doesn't make much difference, although if they were optional and expected not to appear if zero then it would simplify the display logic a bit and we wouldn't need separate boolean properties.

A missing giftAidAmount feels a bit more correct than a zero amount when the donor has said they don't want to claim gift aid.